### PR TITLE
Gokay finished the test case of US 500  Test case 1

### PR DESCRIPTION
**Pull request by gokayyazar**
added this code here
added this code there

### CODE REVIEW

**Review by gozelihsan — COMMENTED** [review]
great code!

File: src/test/java/Gokay/Gokay.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Gokay;
+
+public class Gokay {
+    public static void main(String[] args) {
+        System.out.println("hello team");
```
Comment (kgasimov) [inline src/test/java/Gokay/Gokay.java:5]: Please can you make changes in this line, please start with upper case

**Review by kgasimov — CHANGES_REQUESTED** [review]
Hi Gokay, your codes looks good. Please can you make changes on line 5

**Review by gokayyazar — COMMENTED** [review]
already done

**Comment by klevismuda** [comment]
Perfect

**Review by gozelihsan — APPROVED** [review]
great

**Review by kgasimov — APPROVED** [review]
Great Job, brother!

**Pull request merged**